<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>650</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>650</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>DESAnalysis</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="tab1">
      <attribute name="title">
       <string>#SBoxDifference</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="tab1_line1">
         <item>
          <widget class="QLabel" name="labInDiff">
           <property name="text">
            <string>InDiff :</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="boxInDiff"/>
         </item>
         <item>
          <widget class="QLabel" name="labSBoxId">
           <property name="text">
            <string>SBoxId : </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="boxSBoxId"/>
         </item>
         <item>
          <widget class="QPushButton" name="btnInDiffGo">
           <property name="text">
            <string>Go!</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTableWidget" name="tableWidget">
         <column>
          <property name="text">
           <string>OutDiff</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Number</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>ProbableInput</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab2">
      <attribute name="title">
       <string>#DigitChange</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="tab2_line1">
         <item>
          <widget class="QLabel" name="labPlain">
           <property name="text">
            <string>Plain Text:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="edtPlain">
           <property name="toolTip">
            <string/>
           </property>
           <property name="statusTip">
            <string/>
           </property>
           <property name="whatsThis">
            <string/>
           </property>
           <property name="inputMask">
            <string/>
           </property>
           <property name="text">
            <string notr="true">PLAIN123</string>
           </property>
           <property name="maxLength">
            <number>8</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="labKey">
           <property name="text">
            <string>Key:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="edtKey">
           <property name="text">
            <string notr="true">KEY98765</string>
           </property>
           <property name="maxLength">
            <number>8</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="tab2_line2">
         <item>
          <widget class="QLabel" name="labChangeDigit">
           <property name="text">
            <string>ChangeDigit</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="boxChangeDigit"/>
         </item>
         <item>
          <widget class="QRadioButton" name="btnChangePlainText">
           <property name="text">
            <string>ChangePlainText</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="btnChangeCipherText">
           <property name="text">
            <string>ChangeCipherText</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnChangeGo">
           <property name="text">
            <string>Go</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QCustomPlot" name="plotWidget" native="true"/>
       </item>
       <item>
        <widget class="QProgressBar" name="progressBar">
         <property name="value">
          <number>24</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab3">
      <attribute name="title">
       <string>About</string>
      </attribute>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>./include/qcustomplot/qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
